py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -368,6 +368,8 @@ def __init__(self, remote_server_addr, keep_alive=False, resolve_ip=True):
                 ('POST', '/session/$sessionId/window/rect'),
             Command.GET_WINDOW_RECT:
                 ('GET', '/session/$sessionId/window/rect'),
+            Command.W3C_MINIMIZE_WINDOW:
```

Comment:
The appveyor build fail is not due to your changes. I would prefer you update this PR with changes as opposed #4272 as that one would require more changes.